### DISCUSSION TITLE
empty response with latest version

### DISCUSSION THREAD

**Discussion by codeman5000** [lexiforest/curl_cffi#347] [Q&A]
Hello,

i noticed indeed requests dit not work anymore with last version 0.6. so i upgraded to last version 0.7.
however on a specific endpoint i do requests to i get all empty responses. i get empty lines. previously when the fingerprints still worked in version 0.6 everything worked great.

Is there a way i can send u a private message with the link and see what response you get?
the url i do the request to is kinda hidden and secret. i do not like to share it here.

kind regards.

**Comment by perkfly** [comment]
Please find my email address on my profile page.
